invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -129,3 +129,74 @@
 .item div.description:not(.license-item-description) {
   display: initial !important;
 }
+
+/* Styling for the community selection modal */
+
+.community-selection-modal {
```

Comment:
did you check if all the custom CSS can be avoided?

Reply:
Was able to get rid of a piece :+1: